app/models/bulk_submission.rb

```diff
@@ -324,6 +324,7 @@ def extract_request_options(details) # rubocop:todo Metrics/CyclomaticComplexity
       end
       request_options['gigabases_expected'] = details['gigabases expected'] if details['gigabases expected'].present?
```

Comment:
I think this can be tidied up a bit. Sure there must be a ruby or rails method which could avoid the repetition?

Reply:
Yes, refactoring that